test/proposal/lvm_helper_test.rb

```diff
@@ -307,7 +307,7 @@
 
         expect(lvs).to contain_exactly(
           an_object_with_fields(lv_name: "one", size: (9.GiB - 4.MiB).to_i),
-          an_object_with_fields(lv_name: "two", size: 9.GiB.to_i),
+          an_object_with_fields(lv_name: "two", size: (9.GiB - 4.MiB).to_i),
```

Comment:
Yes, there was a bug (that was, in fact, the bug I wanted to correct). The sum of the LVs was bigger than the VG size. I have added another commit to make it more obvious in the test. Obviously, better checks in libstorage-ng would have prevented us to introduce the bug. But that's a WIP.